extensions/exchanges/kraken/exchange.js

```diff
@@ -1,352 +1,362 @@
 var KrakenClient = require('kraken-api'),
```

Comment:
Did you mean to completely redo the indenting of this file?

Reply:
That's my fault on last commit on exchange.js on my fork, but this commit is not included in this pull request. Only update-products.sh and products.json should be merged.

Anyway, if you are interested in my fix for kraken connection error I'll fix the indenting asap

Reply:
Sure am, so please do :)

Reply:
ok :) this evening. I'll ping you when done